packages/back-end/src/models/DataSourceModel.ts

```diff
@@ -90,6 +90,22 @@ export async function getDataSourcesByOrganization(
     context.permissions.canReadMultiProjectResource(ds.projects)
   );
 }
+
+// WARNING: This does not restrict by organization
+export async function _dangerourslyGetAllDatasourcesByOrganizations(
+  organizations: string[]
+): Promise<DataSourceInterface[]> {
+  if (usingFileConfig()) {
+    return getConfigDatasources(organizations[0]);
```

Comment:
Is this a valid assumption that `organizations` will only have one org when using file config? Or does this need to loop through all the orgs in the list

Reply:
good catch.  This doesn't belong here as it doesn't apply to cloud, and this is only used on cloud, I've gotten rid of it.